docs/general/ddl/data-types.rst

```diff
@@ -668,6 +703,7 @@ For example::
     +-------------------------------+
     | result                        |
     +-------------------------------+
+    +-------------------------------+
```

Comment:
Mistake? ;)

Reply:
yes, conflicts resolving mistake, thanks for spotting it.